assessment.md

```diff
@@ -202,39 +202,78 @@ TODO: add description for each assessment level for this skill.
+  * Good: Student does part and shows up and communicates effectively
+  * Excellent: Student does part and works effectively and efficiently with group members. Communications effectively
+
 
 * Conduct
 
-TODO: add description for each assessment level for this skill.
+  * None: Student physically attacks other students and yells at them, insults and acts rudely towards other students. Sabotages the project
+    if student in group does not get their way
+  * Inadequate: Student tries to pick a fight with everyone, acts rudely and is unaccepting of students. Refuses to work with others.
+  * Adequate: Student follows rules and works with everyone and is respectful.
+  * Good: Student is kind and respectful. Openminded and willing to learn. Works with everyone.
+  * Excellent: Student kind and respectful. Openminded and eager to learn. Works with everyone and is helpful
+
 
 * Decisions
 
-TODO: add description for each assessment level for this skill.
+  * None: Makes decisions without consulting the rest of the group.
+  * Inadequate: Consults the group, but still makes their own decisions.
+  * Adequate: Consults the group to come to a consensus of what decision should be made.
+  * Good: Consults the group to come to a consensus of what decision should be made, whether or not they believe it to be the best answer
+  * Excellent: Consults the group and contributes to a healthy conversation to come to a consensus of what decision should be made, whether or not they believe it to be the best answer, and supports it.
 
 * Postmortems
 
-TODO: add description for each assessment level for this skill.
+  * None: Does not contribute when fixing remaining issue and blames other group members.
+  * Inadequate: Unwilling to fix postmortems in a proper manner, and isn't helping the team.
+  * Adequate: Subtly helps the group fix remaining issues.
+  * Good: Helps the group fix multiple issues, and contributes a lot.
+  * Excellent: Leads the group, and guides everyone to fix the issues as a team.
 
 * Conflicts
 
-TODO: add description for each assessment level for this skill.
+  * None: When a conflict arises, student does not acknowledge finding a resolution with the other(s) and the conflict affects the work.
+  * Inadequate: When a conflict arises, student talks through it, but is unable to find a resolution.
+  * Adequate: When a conflict arises, student talks through it and and a sub-par resolution is achieved.
+  * Good: When a conflict arises, student talks through it and finds a solution.
+  * Excellent: When a conflict arises, student talks through it, finds a solution with the other students, and appologizes for the mishap.
 
 * Infractions
 
-TODO: add description for each assessment level for this skill.
+  * None: 5th time = fail
+  * Inadequate:4th time = overall grade reduction
+  * Adequate:3rd time = grade reduction for assignment
+  * Good: 2nd time = talk with professor
+  * Excellent:1nd time = a warning
```

Comment:
I think you should clarify what you mean by `time`. You could say that `Student has incurred x number of infractions` or something along these lines